games/f.yaml

```diff
@@ -618,6 +618,27 @@
   updated: 2022-09-01
   info: FreeCS is a free-software reimplementation of Counter-Strike 1.5 running on the FTEQW engine.
 
+- name: FreeHeroes
+  development: active
```

Comment:
```suggestion
  development: sporadic
```

Reply:
No objections from me, it's more accurate